sybilMTP.go

```diff
@@ -0,0 +1,243 @@
+package circuits
+
+import (
+	"encoding/json"
+	"fmt"
+	core "github.com/iden3/go-iden3-core"
+	"github.com/iden3/go-merkletree-sql/v2"
+	"github.com/pkg/errors"
+	"math/big"
+	"strconv"
+)
+
+type SybilMTPInputs struct {
+	BaseConfig
+
+	ID                       *core.ID `json:"id"`
+	ProfileNonce             *big.Int `json:"profileNonce"`
+	ClaimSubjectProfileNonce *big.Int `json:"claimSubjectProfileNonce"`
+
+	IssuerClaim ClaimWithMTPProof `json:"issuerClaim"`
+	HolderClaim ClaimWithMTPProof `json:"holderClaim"`
+
+	GISTProof GISTProof `json:"gistProof"`
+	CRS       *big.Int  `json:"crs"`
+
+	RequestID *big.Int `json:"requestID"`
+	IssuerID  *core.ID `json:"issuerID"`
```

Comment:
Do you really need IssuerID? 
You have the IssuerID inside ClaimWithMTPProof, maybe it is redundant.

Reply:
Its redundant, Im updating it thanks!